src/routes/tools/local-climate-change-snapshot/explore.svelte

```diff
@@ -173,19 +188,30 @@
   async function initApp() {
     const { query } = $page;
     // Get initial configuration (from default or from url)
-    const { lat, lng, boundary, indicator, imperial } = getInitialConfig(
-      query,
-      DEFAULT_INITIAL_CONFIG
-    );
+    let {
+      lat,
+      lng,
+      boundary: boundaryType,
+      indicator,
+      imperial,
+    } = getInitialConfig(query, DEFAULT_INITIAL_CONFIG);
+    lat = +lat;
+    lng = +lng;
     // Set intial values for stores
+    if (!$location || !$boundary) {
+      let loc;
+      try {
+        loc = await getLocationFromQuery(lng, lat, boundaryType, true);
+      } catch (error) {
+        console.warn(error);
```

Comment:
Should maybe display a warning here via the notify as if the location isn't correctly set the app won't load.